test/versioned/mcp-sdk/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "mcp-sdk-tests",
+  "targets": [{"name":"@modelcontextprotocol/sdk","minAgentVersion":"13.0.0"}],
+  "version": "0.0.0",
+  "private": true,
+  "engines": {
+    "node": ">=20"
+  },
+  "tests": [
+    {
+      "engines": {
+        "node": ">=20"
+      },
+      "dependencies": {
+        "@modelcontextprotocol/sdk": ">=1.13.0",
+        "zod":  "^3.23.8"
```

Comment:
by having this here, it's going to test diff versions of this package as well. I don't see this being necessary